### DISCUSSION TITLE
Need to delete Storybook github issue

### DISCUSSION THREAD

**Discussion by avalery** [storybookjs/storybook#26790] [Help] [answered]
### Summary

Hello,
My company realized that the url of one of its internal development environment has been communicated in a Storybook github issue: https://github.com/storybookjs/storybook/issues/12086
The issue was created by a former employee. Could you please delete this ticket, or remove the private information that is inside? If required, I can ask this person to give his agreement.
Thank you

**Comment by valentinpalkovic** [comment, marked as answer]
I have deleted the issue since the history is preserved even after editing it.
